Misc/NEWS.d/next/Library/2020-11-15-15-23-34.bpo-42345.hiIR7x.rst

```diff
@@ -0,0 +1,4 @@
+Fix ``typing.Literal`` equals method to ignore the order of arguments.
+Fix issue related to ``typing.Literal`` caching by adding ``typed``
+parameter to ``typing._tp_cache`` function. Add deduplication of
+``typing.Literal`` arguments. Patch provided by Yurii Karabas.
```

Comment:
I would rewrite this as a single sentence, e.g. "Fix various issues with typing.Literal parameter handling (flatten, deduplicate, use type to cache key)."

Reply:
Thanks,  it sounds much cleaner.